pkg/metrics/metrics.go

```diff
@@ -372,6 +408,13 @@ func (m *ServerMetrics) InitSchedule(scheduleName string) {
 	// -------------------------------------------------------------------
 }
 
+// RegisterBackup records the existing number of backups
+func (m *ServerMetrics) RegisterBackup(backupSchedule string) {
```

Comment:
I'm wondering if we want these particular metric to be scoped to schedules. Since they're meant to represent all the backups (or restores) present, getting an accurate count would require subtracting the deletion metric from the number of backups per schedule, then summing all of those values.

This has the drawback of being more work on the scraping side, but the benefit of getting a fine-grained count of backups/restores per-schedule. What are your thoughts, @fabito?